src/qtfirebaseadmob.h

```diff
@@ -504,28 +503,21 @@ class QtFirebaseAdMobRewardedVideoAd : public QtFirebaseAdMobBase, public fireba
     Q_OBJECT
 
 public:
```

Comment:
I'd suggest we keep seperate ENUM types for each interstitial type- this way we don't need to refactor again should Google add more (different) types